internal/provider/resources/resource_deployment_test.go

```diff
@@ -103,10 +103,11 @@ func TestAcc_ResourceDeploymentHybrid(t *testing.T) {
 			},
 			// Import existing deployment and check it is correctly imported - https://stackoverflow.com/questions/68824711/how-can-i-test-terraform-import-in-acceptance-tests
 			{
-				ResourceName:            resourceVar,
-				ImportState:             true,
-				ImportStateVerify:       true,
-				ImportStateVerifyIgnore: []string{"external_ips", "oidc_issuer_url"},
+				ResourceName:      resourceVar,
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateVerifyIgnore: []string{"external_ips", "oidc_issuer_url", "image_version", "scaling_status.hibernation_status.%",
+					"scaling_status.hibernation_status.is_hibernating", "scaling_status.hibernation_status.reason"},
```

Comment:
the results may be different between the state and what the API returns but since these values are `computed` then it doesnt really matter what the values are for test purposes
ie. we don't want the test to fail if these values are different since they are not altered by terraform in any way